jira.xml

```diff
@@ -41,6 +41,24 @@
     <issue>
       Sep 20th: SPIKE Onboarding.
     </issue>
```

Comment:
Yeah, we have an onboarding session.

It will be in Turkish, but international audiences are always welcome.

I can switch to English whenever needed, and the Turkish audience adqequately understands English too.

Here's a registration form, for the interesed :) : 

https://app.youform.com/forms/cqfm7reo